keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -714,12 +795,19 @@ message ProvisionerRevokeBucketAccessResponse {
 # Graduation Criteria
 
 ## Alpha
-
 - API is reviewed and accepted
 - Implement all COSI components to support Greenfield, Green/Brown Field, Brownfield and Static Driverless provisioning
 - Evaluate gaps, update KEP and conduct reviews for all design changes 
 - Develop unit test cases to demonstrate that the above mentioned use cases work correctly
 
+## Alpha -> Beta
+- Basic unit and e2e tests as outlined in the test plan.
+- Metrics in k/k for bucket create and delete, and granting and revoking bucket access.
+- Metrics in provisioner for bucket create and delete, and granting and revoking bucket access.
+
+## Beta -> GA
+- Stress tests to iron out possible race conditions in the controllers.
+- Users deployed in production and have gone through at least one K8s upgrade.
```

Comment:
Is there a plan for reusable tests that can be run against implementations (e.g., MinIO, Rook-Ceph, Amazon S3, etc.) to ensure all functionality is implemented as required by COSI and common bugs are handled?

Reply:
We started discussing this, and whilst AFAIK there may be no 'plan' as of now, there is such intent.